### PR TITLE
refactor: wallet tab components

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary
Please summarize your changes, describing __what__ they are and __why__ they were made.

### Changelog
```
Please write an accurate and concise changelog for your changes.
```

## Relevant Issues
Please list any related issues (e.g. bug, task).

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/index.js [hunk 1]
```diff
@@ -78,4 +77,9 @@ export { default as StakingAssetTile } from './StakingAssetTile.svelte'
 // atomic components
 export * from './atoms'
 export * from './molecules'
+export * from './organisms'
+
 export * from './inputs'
+
+// panels
```
Comment (maxwellmattryan) [inline packages/shared/components/index.js:84]: Is this comment supposed to be panes or panels?

File: packages/shared/components/organisms/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export { default as AccountAssetsList } from './AccountAssetsList.svelte'
+export { default as AccountSummary } from './AccountSummary.svelte'
+export { default as AccountHistory } from './AccountHistory.svelte'
```
Comment (maxwellmattryan) [inline packages/shared/components/organisms/index.js:3]: Can these be alphabetized?

File: packages/shared/components/panes/wallet/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export { default as AccountSummaryAndAssetsPane } from './AccountSummaryAndAssetsPane.svelte'
```
Comment (maxwellmattryan) [inline packages/shared/components/panes/wallet/index.js:1]: Can these be alphabetized?

File: packages/shared/routes/dashboard/wallet/Wallet.svelte [hunk 4]
```diff
@@ -1,47 +1,37 @@
...
     import { initialiseListeners } from 'shared/lib/walletApiListeners'
     import { onMount } from 'svelte'
-    import { AccountAssets, AccountBalance, AccountHistory, BarChart, LineChart, ManageAccount, Send } from './views/'
 
-    const { accounts, accountsLoaded, internalTransfersInProgress } = $wallet
-
-    let modal: Modal
+    const { accounts, accountsLoaded } = $wallet
 
+    // TODO: move to dashboard or lib
```
Comment (maxwellmattryan) [inline packages/shared/routes/dashboard/wallet/Wallet.svelte:34]: There's quite a lot of TODO comments in the code already, so we probably shouldn't add anymore. We can make a task for later?

**Review by maxwellmattryan — CHANGES_REQUESTED** [review]
Code overall looks quite good; much cleaner than before :pray:. Have requested some small minor adjustments.

File: packages/shared/routes/dashboard/wallet/Wallet.svelte (hunk 4 above)
Comment (nicole-obrien) [reply packages/shared/routes/dashboard/wallet/Wallet.svelte:34]: Yep I will leave the deep comment in and make a task too. Lots of todos are already contained with in larger tasks on backlog. And stardust develop is very much a work in progress branch until we finish a lot of refactoring.

File: packages/shared/components/organisms/index.js (hunk 2 above)
Comment (nicole-obrien) [reply packages/shared/components/organisms/index.js:3]: Done

**Review by maxwellmattryan — APPROVED** [review]

**Pull request merged**